src/Uno.UI.Runtime.Skia.Wpf/Controls/WpfTextViewTextBox.cs

```diff
@@ -3,12 +3,12 @@
 
 namespace Uno.UI.Runtime.Skia.Wpf.Controls;
 
-internal class WpfTextViewTextBox : TextBox
+internal class WpfTextViewTextBox2 : TextBox
```

Comment:
Oh this **was** public in past. I will find another public type to test the change on.